Assets/Scripts/Accelerometer.cs

```diff
@@ -1,35 +1,45 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 
 public class Accelerometer : MonoBehaviour
 {
+    private Vector2 _velocity;
```

Comment:
Please use the public property with a private setter.

Reply:
Done: https://github.com/cjh15hub/terribletweeters/pull/1/commits/bf5906843a21d2e978aa0e82765b2f23f2a3315e#diff-b9a61da49fb7ea93aad54b4b02c2a551caa0e1f74721482088ba4ee4d1a269e6R7